Added Contact Form

BertrandConxy opened the pull request:
Hello 👋 

## Changes made in this repository:
 During Pair programming with my partner Yerimah, we have built a contact form, and here are things in it

- contact form is in the footer tag as we have to use semantic tags as much as possible for easy accessibility.
- The form has three inputs
- For the full name input there is a limited number of characters of 30 characters and it has the required attribute as it's in the instructions
- For email input, it has the type of email just for email validation and it is required too
- For message input, it has the number of a maximum character of 500 and it is required too.
- we have added submit button which will submit the data to form a database on Formspree
- All elements are laid out using Flexbox
- There are also social icons.

I look forward to your best and swift review. Thanks

BertrandConxy committed:
Add contact form skeleton

BertrandConxy committed:
Adding footer

Yerimah committed:
add styles to the contact form

BertrandConxy committed:
Correct some styling in contact form

BertrandConxy committed:
Add layout of Flexbox to contact form in footer

MrBrN197 commented on index.html:
```diff
@@ -166,8 +167,36 @@ <h3>Skills</h3>
+            
+           <textarea id="message" name="message" placeholder="Enter text here" maxlength="500" required></textarea>
+            
+           <input type="submit" id="submit" value="Get in touch">
```
- [ ] Hi, everything else about your design is really good 👍. In order to have this template match the Figma perfectly please ensure that you implement the interaction state correctly as per the Figma template. 
please implement the `:active` state for this button
## Your Design
![image](https://user-images.githubusercontent.com/18730106/145086115-e7fcb867-958f-4675-851f-80d278e664de.png)
## Figma Design (Interaction states)
![image](https://user-images.githubusercontent.com/18730106/145086211-bdd69816-b435-4083-b4ba-d8e3d2c96344.png)

MrBrN197 commented on index.html:
```diff
@@ -166,8 +167,36 @@ <h3>Skills</h3>
+            I'm always interested in hearing about new projects,
+             so if you'd like to chat please get in touch.</p>
+
+        <form action="https://formspree.io/f/xpzbzzjw" method="POST">
```
- [ ] Your design is working well 👍; the requirements want you to use `flexbox` for this section. please layout these elements using `flexbox`. this will help you practice using `flexbox`.

MrBrN197 reviewed (changes requested):
## Hi, @BertrandConxy & @Yerimah  👋 

You've done a really nice job so far.
There are some issues that you still need to work on to go to the next project but you are almost there!
## To highlight 👏👏
- ✔Your design matches the Figma well.
- ✔Professional README

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BertrandConxy committed:
Implemented button interactions like : active to contact form buttons and also used layout flexbox to the form element

BertrandConxy commented:
Thank you @MrBrN197 for your comments.

## In recent commits I added the following:
- Implemented button interaction as it in the Figma template design that I am working on.
![button interaction](https://user-images.githubusercontent.com/90222110/145185213-cc54c33b-3249-4596-b3ce-d91338fcc195.jpg)
-I also added a display of flex in the Form element styles so that the inputs are laid out using flexbox
![form flex](https://user-images.githubusercontent.com/90222110/145185438-4ea0d102-9b48-451f-8b99-92088e22ef64.jpg)

BertrandConxy committed:
Add align content center to footer

ignatius22 reviewed (approved):
## Hi @BertrandConxy, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
